src/Atmos/Model/radiation.jl

```diff
@@ -54,12 +55,12 @@ function flux_radiation!(m::StevensRadiation, flux::Grad, state::Vars,
   z = aux.orientation.Φ/grav
   Δz_i = max(z - m.z_i, -zero(FT))
   # Constants
-  cloud_top_cooling = m.F_0 * exp(-aux.∫dnz.radiation.∂κLWP)
+  cloud_top_cooling  = m.F_0 * exp(-aux.∫dnz.radiation.∂κLWP)
   cloud_base_warming = m.F_1 * exp(-aux.∫dz.radiation.∂κLWP)
   free_troposphere_cooling = m.ρ_i * FT(cp_d) * m.D_subsidence * m.α_z * ((cbrt(Δz_i))^4 / 4 + m.z_i * cbrt(Δz_i))
-  F_rad = cloud_base_warming + cloud_base_warming + free_troposphere_cooling
-  flux.ρe += SVector(FT(0), 
-                     FT(0), 
+  F_rad = cloud_base_warming + cloud_top_cooling + free_troposphere_cooling
+  flux.ρe -= SVector(FT(0),
```

Comment:
Better (because it works irrespective of geometry):
```suggestion
  flux.ρe += F_rad .* vertical_unit_vector 
```

Reply:
I'll change once #609 lands, since `vertical_unit_vector` was defined in #609.

Reply:
I'll change once #609 lands, since vertical_unit_vector was defined in #609.